src/Pos/Explorer/DB.hs

```diff
@@ -28,6 +30,17 @@ import           Pos.Txp.Toil          (Utxo)
 import           Pos.Util.Chrono       (NewestFirst (..))
 
 ----------------------------------------------------------------------------
+-- Types
+----------------------------------------------------------------------------
+
+type Page = Integer
+type Epoch = Integer
```

Comment:
I don't like to wrap and unwrap code too much, the client is sending an `Integer`, so I decided that `Integer` is the most simple thing to do. I can change it if you think `EpochIndex` is better.